packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -1,80 +1,173 @@
-import { Avatar, Box, Flex, Text } from '@audius/harmony'
+import { Avatar, Box, Flex, IconCamera, IconImage, Text } from '@audius/harmony'
 
 import {
   getCoverPhotoField,
-  getHandleField,
   getNameField,
   getProfileImageField
 } from 'common/store/pages/signon/selectors'
 import { useMedia } from 'hooks/useMedia'
 import { useSelector } from 'utils/reducer'
 
-export const AccountHeader = () => {
+import { ImageField } from './ImageField'
+
+type AccountHeaderProps = {
+  mode: 'editing' | 'viewing'
+}
+
+const CoverPhotoBox = ({
+  imageUrl,
+  isEditing
+}: {
+  imageUrl: string | undefined
+  isEditing?: boolean
+}) => (
+  <Box
+    h='100%'
+    w='100%'
+    border='default'
+    css={{
+      backgroundColor: 'lightgray',
+      overflow: 'hidden',
+      ...(imageUrl
+        ? {
+            backgroundImage: `url(${imageUrl})`,
+            backgroundPosition: 'center',
+            backgroundSize: '100%',
+            backgroundRepeat: 'no-repeat, no-repeat'
+          }
+        : {
+            background:
+              'linear-gradient(0deg, rgba(0, 0, 0, 0.20) 0%, rgba(0, 0, 0, 0.00) 100%), #C2C0CC'
+          })
```

Comment:
Is this only supposed to be for when the photo is empty? For some reason I thought cover photos always had a mild shadow gradient on them

Reply:
Checked with Sammie and you are correct! updated